_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -2897,11 +2897,11 @@
 	pixel_x = -3;
 	pixel_y = 3
 	},
-/obj/item/cigbutt/cigarbutt{
```

Comment:
rip cigar butt and everything else that got removed instead of moved around

Reply:
I believe I only removed that cigar butt, two blue folders, and a blank paper.